src/test/java/org/broadinstitute/hellbender/tools/IndexFeatureFileIntegrationTest.java

```diff
@@ -265,6 +265,61 @@ public void testBedIndex() {
         Assert.assertEquals(index.getSequenceNames(), Arrays.asList("1", "2", "4"));
     }
 
+    // TODO: this is not enabled because canDecode returns false for bed.gz extension
+    // TODO: should this be changed at the htsjdk level to return true also for a block-compressed extension?
+    @Test(enabled = false)
+    public void testBedGZIndex() {
+        final File ORIG_FILE = getTestFile("test_bed_for_index.bed.gz");
+        final File outName = BaseTest.createTempFile("test_bed_for_index.bed.", TabixUtils.STANDARD_INDEX_EXTENSION);
+
+        final String[] args = {
+                "--feature_file" ,  ORIG_FILE.getAbsolutePath(),
+                "-O" ,  outName.getAbsolutePath()
+        };
+        final Object res = this.runCommandLine(args);
+        Assert.assertEquals(res, outName.getAbsolutePath());
+
+        final Index index = IndexFactory.loadIndex(res.toString());
+        Assert.assertTrue(index instanceof LinearIndex);
```

Comment:
Sorry, copy-paste error. Fixed while extracting the common code from both tests.